### PR TITLE
feat: create user repository and edit schema

### PR DESCRIPTION
**Pull request by Nhsdkk**
1) Create user repository
2) Add default values to db schema
3) Add converters to User class
4) Delete typing on singleton decorator, because it causes unexpected behavior

### CODE REVIEW

**Comment by Nhsdkk** [comment]
5) Add ValueNotFoundError and UniqueError

**Review by MaxPlays35 — APPROVED** [review]
Approved

File: identity_service/repository/user_repository.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+class UserRepository:
+    """
+    Data class that stores user information
+
+    Attributes
+    ----------
+    _db_client : prisma.Client
+        Postgres db client
+
+
```
Comment (ZixMai) [inline identity_service/repository/user_repository.py]: Remove empty string

File: identity_service/src/models/user.py [hunk 2]
```diff
@@ -46,3 +53,51 @@ def to_grpc_user(self) -> GrpcUser:
...
+        Parameters
+        ----------
+        exclude: Optional[List[str]]
+            Fields to exclude. All field names should be exactly the same as class attribute
+
+        Returns
+        -------
+        dict
+            User data represented in dictionary
+
```
Comment (ZixMai) [inline identity_service/src/models/user.py]: Remove empty string :D

File: identity_service/db/db.py
Comment (ZixMai) [inline identity_service/db/db.py:1]: Maybe remove some empty strings...

**Review by ZixMai — CHANGES_REQUESTED** [review]

File: identity_service/repository/user_repository.py (hunk 1 above)
Comment (Nhsdkk) [reply identity_service/repository/user_repository.py]: fixed

File: identity_service/src/models/user.py (hunk 2 above)
Comment (Nhsdkk) [reply identity_service/src/models/user.py]: fixed

File: identity_service/db/db.py
Comment (Nhsdkk) [reply identity_service/db/db.py:1]: fixed

**Review by ZixMai — APPROVED** [review]

**Pull request merged**